### PR TITLE
feat(evm): Support EVM deployment with Madara

### PR DESCRIPTION
**Pull request by antiyro**
This pull request adds the necessary environment to easily deploy an EVM sequencer (devnet) or a specific full node with Kakarot v0.9.2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ RUN cargo build --release
 FROM debian:bookworm
 # Install runtime dependencies
 RUN apt-get -y update && \
-    apt-get install -y openssl ca-certificates &&\
+    apt-get install -y openssl ca-certificates tini &&\
```
Comment (Trantorian1) [inline Dockerfile:29]: This is not necesasry as we do not use tine currently. Either remove it or consider updating the Dockerfile as per [this guide](https://github.com/krallin/tini?tab=readme-ov-file#using-tini)

File: evm/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,145 @@
+version: "3.8"
```
Comment (Trantorian1) [inline evm/docker-compose.yml]: Any way to test locally that this works?

**Review by Trantorian1 — CHANGES_REQUESTED** [review]

File: evm/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      context: ../
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    ports:
+      - 9944:9944
+    networks:
+      - internal_prod
+    volumes:
+      - madara_files_prod:/var/lib/madara
+    entrypoint: ["tini", "--"]
```
Comment (antiyro) [inline evm/docker-compose.yml:13]: here @Trantorian1

File: Dockerfile (hunk 1 above)
Comment (antiyro) [reply Dockerfile:29]: the evm compose uses it

Comment (jbcaron) [reply Dockerfile:29]: it would be good to set it as the entrypoint

File: evm/docker-compose.yml (hunk 2 above)
Comment (jbcaron) [reply evm/docker-compose.yml]: this definition is no longer necessary

**Comment by antiyro** [comment]
I will implement configuration file to improve user experience on EVM configuration

**Comment by antiyro** [comment]
Force merging this since it doesnt affect the core Madara

**Pull request merged**